app/controllers/tweets_controller.rb

```diff
@@ -48,6 +48,18 @@ def update
     end
   end
 
+  # DELETE /tweets/:id
+  def destroy
+    @tweet = Tweet.find(params[:id])
```

Comment:
そもそもログインしていないユーザーのツイートは取得しないようにしよう。

```suggestion
    @tweet = current_user.tweets.find(params[:id])
```